tests/test_zipfs.py

```diff
@@ -224,3 +226,38 @@ class TestOpener(unittest.TestCase):
     def test_not_writeable(self):
         with self.assertRaises(NotWriteable):
             open_fs("zip://foo.zip", writeable=True)
+
+
+class FSWithoutMtime(MemoryFS):
+    '''MemoryFS subclass that doesn't return details namespace
+    '''
+    def getinfo(self, path, namespaces):
+        if namespaces is not None:
+            namespaces = set(namespaces) - {'details'}
+        return super().getinfo(path, namespaces)
+
+
+class TestZipFSMtimeFallback(unittest.TestCase):
+    def setUp(self):
+        fh, self._temp_path = tempfile.mkstemp()
```

Comment:
Would it be "better" to use a [TempFS](https://docs.pyfilesystem.org/en/latest/reference/tempfs.html) here? :shrug: